dependencies.yaml

```diff
@@ -84,8 +84,12 @@ dependencies:
             packages:
               - python=3.12
           - matrix:
+              py: "3.13"
             packages:
-              - python>=3.10,<3.13
+              - python=3.13
+          - matrix:
+            packages:
+              - python>=3.10,<3.14
```

Comment:
```suggestion
              - python>=3.10,<3.14a0
```

Reply:
@jakirkham Do you have thoughts on how to coordinate this suggestion with the proposal to drop all alpha specs? https://github.com/rapidsai/build-planning/issues/144

I am inclined to say we should leave this as-is, but I don’t have strong feelings.

Reply:
Suppose we could have a different pin for conda & pip

That said, given one needs to opt-in to use RCs from conda-forge. It doesn't seem like that big of a risk